data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v1.0.xml

```diff
@@ -166,9 +166,6 @@
             <column name="leaf_idx" type="INT">
                 <constraints nullable="false"/>
             </column>
-            <column name="is_relevant" type="BOOLEAN">
-                <constraints nullable="false"/>
-            </column>
```

Comment:
So we don't store it this way, but where is the schema for storing the relevancy information in the new format? What did I miss?

Reply:
Reading the code - do we reuse the tx output table to only record relevant states in it?

Reply:
My understanding was that we only insert into table `utxo_transaction_relevancy` when component is relevant, in which case this field would always be `true`, isn't that the case?

Reply:
Ok, that makes sense.

Reply:
> Reading the code - do we reuse the tx output table to only record relevant states in it?

Tx output table is not used at the moment.